index.js

```diff
@@ -40,6 +40,7 @@ module.exports = (app) => {
       context,
       branch,
       lastRelease,
+      config
```

Comment:
According to style it should be 😅 
```suggestion
      config,
```

Reply:
Doh! yeah, I disabled the hook!